generated/nidaqmx/_task_modules/ai_channel_collection.py

```diff
@@ -57,6 +56,79 @@ def _create_chan(self, physical_channel, name_to_assign_to_channel=''):
+            voltage_excit_val=2.5, gage_factor=2.0,
+            nominal_gage_resistance=350.0, poisson_ratio=0.3,
+            lead_wire_resistance=0.0):
+        """
+        Creates channels to measure two-dimensional strain using a
+        rosette strain gage.
+
+        Args:
+            physical_channel (str): Specifies the names of the physical
+                channels to use to create the strain gage virtual
+                channels necessary to calculate the **rosette
+                measurements** channels.
+            rosette_type (nidaqmx.constants.StrainGageRosetteType):
+                Specifies information about the rosette configuration
+                and measurements.
+            gage_orientation (float): Specifies information about the
+                rosette configuration and measurements.
+            rosette_meas_types (List[nidaqmx.constants.StrainGageRosetteMeasurementType]):
+                Specifies information about the rosette configuration
+                and measurements.
+            name_to_assign_to_channel (Optional[str]): Specifies a name
+                to assign to the virtual channel this function creates.
+                If you do not specify a value for this input, NI-DAQmx
+                creates a default channel name.
+            min_val (Optional[float]): Specifies the minimum strain you
+                expect to measure. This value applies to each strain
+                gage in the rosette.
+            max_val (Optional[float]): Specifies the maximum strain you
+                expect to measure. This value applies to each strain
+                gage in the rosette.
+            strain_config (Optional[nidaqmx.constants.StrainGageBridgeType]):
+                Specifies information about the bridge configuration and
+                measurement.
+            voltage_excit_source (Optional[nidaqmx.constants.ExcitationSource]):
+                Specifies information about the bridge configuration and
+                measurement.
+            voltage_excit_val (Optional[float]): Specifies information
+                about the bridge configuration and measurement.
+            gage_factor (Optional[float]): Contains information about
+                the strain gage and measurement.
+            nominal_gage_resistance (Optional[float]): Contains
+                information about the strain gage and measurement.
+            poisson_ratio (Optional[float]): Contains information about
+                the strain gage and measurement.
+            lead_wire_resistance (Optional[float]): Specifies
+                information about the bridge configuration and
+                measurement.
+        Returns:
+            nidaqmx._task_modules.channels.ai_channel.AIChannel:
+
+            Indicates the newly created channel object.
+        """
+        if rosette_meas_types is None:
+            rosette_meas_types = []
+
+        rosette_meas_types = numpy.int32(rosette_meas_types)
```

Comment:
It's not just the docstring that is wrong. This is wrong, too. This should be converting the `.value` of the enumerations. The way to validate your change is to update [this test](https://github.com/ni/nidaqmx-python/blob/a7a3643911fd549cac198c0a2cc8dadd4a8ad8f4/tests/component/_task_modules/channels/test_ai_channel.py#L848) to not create an array of `.value`, just the enumerations themselves.

Reply:
Note that the way this uses `numpy.int32(some_list)` to create a NumPy array is not well documented and `numpy.typing` doesn't allow this behavior. If we touch any code like this, let's update it to use `numpy.array(some_list, dtype=numpy.int32)`.

Reply:
@zhindes  @bkeryan I updated functions.py to have `rosette_meas_types` parameter reflected with correct type, I'm not sure is this correct approach. If it's okay, I will continue to work on scrapigen to update the metadata accordingly.